src/shipping/components/ShippingZonePostalCodes/ShippingZonePostalCodes.tsx

```diff
@@ -75,6 +73,12 @@ const ShippingZonePostalCodes: React.FC<ShippingZonePostalCodesProps> = ({
   const intl = useIntl();
   const classes = useStyles({});
 
+  const getInclusionType = () =>
```

Comment:
Could look nicer and be a bit simpler
```suggestion
  const getInclusionType = () => {
     if(inclusionType) { return inclusionType; }
     return postalCodes[0]?.inclusionType || PostalCodeRuleInclusionTypeEnum.EXCLUDE;
  }
```